optashift-employee-rostering-gwtui/src/main/java/org/optaplanner/openshift/employeerostering/gwtui/client/util/PromiseUtils.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.openshift.employeerostering.gwtui.client.util;
+
+import elemental2.promise.Promise;
+
+public class PromiseUtils {
```

Comment:
I would suggest making this utility non-static and injectable so that it is more easily mocked when we start adding more unit tests.

Reply:
Good suggestion. I'll make this change.

Reply:
...in a future PR, wdyt? :P

Reply:
That's fine by me. I don't consider it a blocking issue.